packages/api/src/derives/ethBridge/eventProof.ts

```diff
@@ -45,6 +46,8 @@ export function eventProof(instanceId: string, api: ApiInterfaceRx): () => Obser
             return {
               eventId: eventProof.eventId.toString(),
               validatorSetId: eventProof.validatorSetId.toString(),
+              blockHash: eventProof.blockHash.toString(),
+              tag: hexToString(eventProof.tag.toString()),
```

Comment:
is this ok if tag is None?

Reply:
will check

Reply:
ya.. it is fine.. tried a small snippet with

```
const a = api.registry.createType('Option<VersionedEventProof>', null);
console.log('a:::', a.toString());
```

It returns - console.log
    a:::